'SIGPIPE': undeclared identifier on Windows

I get the following build error on Windows:
```
>------ Build started: Project: CMakeLists, Configuration: Debug ------
  [1/2] Building CXX object test\CMakeFiles\module-tests.dir\oatpp-libressl\tests.cpp.obj
  FAILED: test/CMakeFiles/module-tests.dir/oatpp-libressl/tests.cpp.obj 
  C:\PROGRA~2\MIB055~1\2019\COMMUN~1\VC\Tools\MSVC\1428~1.293\bin\Hostx64\x64\cl.exe  /nologo /TP -DCERT_CRT_PATH=\"C:/Users/Leo/Source/Repos/oatpp-libressl/test/../utility/cert/test_cert.crt\" -DCERT_PEM_PATH=\"C:/Users/Leo/Source/Repos/oatpp-libressl/test/../utility/cert/test_key.pem\" -I..\..\..\test -I..\..\..\src -I"C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp" -IC:\Users\Leo\Source\Repos\libressl-3.2.4\out\install\x64-Release\include /DWIN32 /D_WINDOWS /W3 /GR /EHsc /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /Fotest\CMakeFiles\module-tests.dir\oatpp-libressl\tests.cpp.obj /Fdtest\CMakeFiles\module-tests.dir\ /FS -c ..\..\..\test\oatpp-libressl\tests.cpp
C:\Users\Leo\Source\Repos\oatpp-libressl\test\oatpp-libressl\tests.cpp(24): error C2065: 'SIGPIPE': undeclared identifier
  ninja: build stopped: subcommand failed.

Build failed.
```

SIGPIPE does not seem to be available on Windows: https://github.com/facebook/buck/issues/1645
These guys just skipped it if they detect windows. I have no idea about networking, is it ok to just add some `#ifdef _WIN32`?

When `std::signal(SIGPIPE, SIG_IGN);` in `test.cpp` is commented out, the build succeeds.

--- a/test/oatpp-libressl/tests.cpp
+++ b/test/oatpp-libressl/tests.cpp
@@ -21,7 +21,9 @@ void runTests() {
   oatpp::libressl::Callbacks::setDefaultCallbacks();
 
   /* ignore SIGPIPE */
-  std::signal(SIGPIPE, SIG_IGN);
+  #if !(defined(WIN32) || defined(_WIN32))
+    std::signal(SIGPIPE, SIG_IGN);
+  #endif
 
   {
 

